aa-sdk/core/src/utils/defaults.ts

```diff
@@ -51,7 +50,7 @@ export const getDefaultSimpleAccountFactoryAddress = (
         case baseGoerli.id:
         case baseSepolia.id:
         case fraxtal.id:
-        case fraxtalSepolia.id:
+        case 2523:
```

Comment:
I'm not gonna put a comment here because we have an item to just delete this all together since we're gonna remove simple account from here